src/components/LoginForm/LoginForm.tsx

```diff
@@ -0,0 +1,162 @@
+import '../../i18n/i18n.ts';
+import { Box, Button, Divider, Grid, TextField, useMediaQuery, useTheme } from '@mui/material';
+import { Field, Formik } from 'formik';
+import { Link } from 'react-router-dom';
+import { LoginFormErrors } from './LoginForm.types';
+import { LoginOptions } from './LoginOptions';
+import { PasswordInput } from '../PasswordInput/PasswordInput';
+import styles from './LoginForm.module.scss';
+import { useTranslation } from 'react-i18next';
+
+export function LoginForm() {
+  const theme = useTheme();
+  const isMobile: boolean = useMediaQuery(theme.breakpoints.down('tablet'));
+  const minLength = 1;
+  const { t } = useTranslation();
+  const validate = (values: { email: string; password: string }) => {
+    const errors: LoginFormErrors = {};
+    if (values.email.length < minLength) {
+      errors.Email = t('loginForm.emailEmpty');
+    }
+    if (values.password.length < minLength) {
+      errors.Password = t('loginForm.passwordEmpty');
+    }
+    return errors;
+  };
+
+  return (
+    <>
+      <Formik
+        enableReinitialize
+        validateOnMount={true}
+        validateOnChange={true}
+        validateOnBlur={true}
+        validate={validate}
+        initialValues={{ email: '', password: '' }}
+        onSubmit={function () {
+          throw new Error('Function not implemented.');
+        }}
+      >
+        {({ isValid }) => {
+          return (
+            <form>
+              <Box
+                padding={0}
+                boxShadow={'0px 0px 30px #ccc'}
+                maxWidth={{
+                  desktop: 400,
+                  mobile: 330,
+                  tablet: 400,
+                }}
+                height={540}
+                alignItems='center'
+                justifyContent='center'
+              >
+                <Grid
+                  container
+                  paddingTop={{
+                    desktop: 8,
+                    mobile: 6,
+                    tablet: 8,
```

Comment:
This is not needed.
you can simplify check to change layout if we reach table+ resolution:
```tsx
const isTablet = useMediaQuery(theme.breakpoints.up('tablet'));

return (
    <Grid
      container
      paddingTop={isTablet ? 8 : 6}

```

**FYI**: if you put this styles into *`.scss`* files then you do not need to worry about eslint-magic-number error 😉